Leave/leave_interface.py

```diff
@@ -53,24 +52,31 @@ async def HandleLeaveReactions(client, payload):
     message = await channel.fetch_message(payload.message_id)
     embed = message.embeds[0]
     
-    status = UI.ParseEmoji(payload.emoji)
-    if status == None:
+    Action = UI.ParseEmoji(payload.emoji)
+    if Action == None:
         return
 
-    if utils.isNotBot(payload.member) and utils.IsAdmin(payload.member) and leave_db.IsLeaveRequestPending(payload.message_id):
+    if not (utils.isNotBot(payload.member) and utils.IsAdmin(payload.member)):
+        return
```

Comment:
Just split this into a bot check and an admin check. Infinitely more readable

Reply:
correct

Reply:
Done